### PR TITLE
Changelog , Readme , cargo.toml

### PR DESCRIPTION
**Pull request by pxp9**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,15 @@
 # Changelog
 
+## 0.7.0 (2022-08-03)
+
+### [#21](https://github.com/ayrat555/fang/pull/21)
+
+- Create a full functional async processing module for fang which uses `tokio` to summon processes that are more lighter than threads.
+- Renaming code Job starting names into Task [#29](https://github.com/ayrat555/fang/pull/29).
+- Exporting serde correctly, here is the [commit](https://github.com/ayrat555/fang/pull/21/commits/cf2ce19c970e93d71e387526e882c67db53cea18).
```
Comment (ayrat555) [inline CHANGELOG.md:9]:
```suggestion
- Export serde correctly - [commit](https://github.com/ayrat555/fang/pull/21/commits/cf2ce19c970e93d71e387526e882c67db53cea18).
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,15 @@
 # Changelog
 
+## 0.7.0 (2022-08-03)
+
+### [#21](https://github.com/ayrat555/fang/pull/21)
+
+- Create a full functional async processing module for fang which uses `tokio` to summon processes that are more lighter than threads.
+- Renaming code Job starting names into Task [#29](https://github.com/ayrat555/fang/pull/29).
```
Comment (ayrat555) [inline CHANGELOG.md:8]:
```suggestion
- Rename Job starting names into Task [#29](https://github.com/ayrat555/fang/pull/29).
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,15 @@
...
+- Exporting serde correctly, here is the [commit](https://github.com/ayrat555/fang/pull/21/commits/cf2ce19c970e93d71e387526e882c67db53cea18).
+- Migrate all blocking module code into async
```
Comment (ayrat555) [inline CHANGELOG.md:10]: let's remove this ;]

**Review by ayrat555 — APPROVED** [review]

File: CHANGELOG.md (hunk 3 above)
Comment (pxp9) [reply CHANGELOG.md]: Solve in [3754904](https://github.com/ayrat555/fang/pull/59/commits/3754904ae18865fb969d9f736237308d3c0934b2)

**Pull request merged**